client/src/Login.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import Cofedlogo from './CoFEDlogo.png';
+import googlelogo from './GoogleLogo.png';
```

Comment:
let's try to use a consistent naming scheme, so rename these to `CofedLogo` and `GoogleLogo`